compiler/compiler.c

```diff
@@ -274,6 +316,12 @@ const struct command commandList[NUMBER_OF_COMMANDS] = {
             .analysisFunction = NULL,
             .translationPattern = ".LSamePicture_{F}:"
         },
+        {
+                .pattern = "deja vu",
+                .usedParameters = 0,
+                .analysisFunction = NULL,
+                .translationPattern = "jmp main"
```

Comment:
FYI: MemeASM only requires a main-function when you create an executable. When you create an assembly file (`-S`) or an object file (`-O`), using this command would result in the generated code having a potentially undefined reference (Since you could link it with something that defines `start`, but not main)